File: LttGate_App/src/app/AccessLog.ts
export class AccessLog {
    Rcdid  : number
    EmployeeId  : string
    LogDate  : any
  LogTime  : any
   TerminalId  :string
    InOut  :string
    constructor(values: Object = {}) {

        Object.assign(this, values);
        
        }

        
        }

File: LttGate.API/Models/AccessLog.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;

namespace LttGate.API.Models
{
    public partial class AccessLog
    {
        [Key]
        public int Rcdid { get; set; }
        public string EmployeeId { get; set; }
        public DateTime LogDate { get; set; }
        public TimeSpan LogTime { get; set; }
        public string TerminalId { get; set; }
        public string InOut { get; set; }
    }
}
